service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillShowAction.kt

```diff
@@ -177,7 +177,15 @@ class BackfillShowAction @Inject constructor(
                         partition.computed_matching_record_count,
                       )
                     }
-                    td("hidden py-5 pl-8 pr-0 text-right align-top tabular-nums text-gray-700 sm:table-cell") { +"""${partition.matching_records_per_minute} #/m""" }
+                    td("hidden py-5 pl-8 pr-0 text-right align-top tabular-nums text-gray-700 sm:table-cell") {
+                      when {
+                        partition.state != BackfillState.RUNNING -> +"-"
+                        !partition.precomputing_done -> +"Computing size..."
```

Comment:
```suggestion
                        !partition.precomputing_done -> +"Computing..."
```

Less text is better for narrow column